src/ui/Questionnaire/Questionnaires.tsx

```diff
@@ -173,17 +174,24 @@ export function Questionnaires({ surveyUnit }: Readonly<{ surveyUnit: SurveyUnit
 
   const latestState = getMostRecentState(surveyUnit);
 
-  const handleOpenModal = () => {
+  const openQuestionnaire = () => {
+    if (!isWebQuestionnaire) {
+      navigate(`/queen/survey-unit/${id}`);
```

Comment:
It must be `/queen/interrogations/${id}` 

But i think, your started branch is outdated with `main`